import.html

```diff
@@ -259,14 +247,21 @@ <h1 class="hero-heading">Add Your Project</h1>
                         <br/><select id="tileServer">
                             <option value="bing">bing</option>
                             <option value="digital_globe">digital_globe</option>
-                            <option value="google">google</option>
+                            <!-- <option value="google">google</option> -->
+							<option value="custom">custom</option>
 
                         </select>
                     </p>
                 </div>
+                <div class="faq-container">
+                    <div class="faq-title">Custom Tileserver URL</div>
+                    <p class="faq-text">Make sure you have permission. Add a custom tileserver URL that uses {z}, {x} and {y} as placeholders and that already includes the api key.
+                        <br/><input type="text" id="customTileServerUrl" style="color:#212121"/>
+                    </p>
+                </div>
                 <div class="faq-container">
                     <div class="faq-title">Enter Description</div>
-                    <p class="faq-text">Enter the name of your project (15 chars max)
```

Comment:
No more name?

Reply:
We still have a name. :) But I seemed as if the text for the field "description" was just copied from the text for the field "name".